pitop/processing/algorithms/ball_detect.py

```diff
@@ -36,40 +36,60 @@ def __init__(self, color):
         }
         self.match_limit = match_limits[color]
 
-        self.center_points = deque(maxlen=DETECTION_POINTS_BUFFER_LENGTH)
-        self.radius = 0
-        self.angle_from_center = None
+        self._center_points_cv = deque(maxlen=DETECTION_POINTS_BUFFER_LENGTH)
+        self._center = None
```

Comment:
What does this actually represent now? Before, this was "the current position of the center of the ball", which was a shorthand for "get latest value in deque". I'm not clear on what this is now and how it is used.

Reply:
center is the user-facing center, where the center of the frame is (0, 0) - same as we use for the line detect stuff. It's way easier to write code for as a user if the middle of the frame is (0, 0) - it helps with face tracking, ball following, line following etc. The deque is a purely internal variable used for drawing the ball contrail